### DISCUSSION TITLE
Improved ramda repl

### DISCUSSION THREAD

**Discussion by mendrik** [ramda/ramda#3421] [Show and tell]
Hi,

I just wanted to share my new ramda repl.
https://github.com/mendrik/ramda-extras

Since my projects are usually in typescript, I spend a lot of time removing and adding types when switching between repl and my projects.

Additionally I utilize ramda-adjunct and purify-ts, which I miss a lot in the official repl.

So I took all of that and glued it together with the awesome monaco editor.
it has types and also auto completion.

Have fun :-)

**Comment by kedashoe** [comment]
Looks fantastic, love the typescript support!
